spec/models/async_transaction/vet360/base_spec.rb

```diff
@@ -45,6 +51,19 @@
       end
     end
 
+    it 'persists the messages from vet360' do
+      # @TODO we need a cassette that doesn't throw an exception but includes messages
+      VCR.use_cassette('vet360/contact_information/email_transaction_status2') do
+        updated_transaction = AsyncTransaction::Vet360::Base.refresh_transaction_status(
+          user,
+          service,
+          transaction2.transaction_id
+        )
+        expect(updated_transaction.metadata.empty?).to eq(false)
+        expect(updated_transaction.metadata.first.is_a?(Vet360::Models::Message)).to eq(true)
```

Comment:
I think we want to ensure that the metadata is a JSON encoded string, not a specific type of ours, but I may be understanding this test incorrectly.